libvast/vast/experimental_table_slice_builder.hpp

```diff
@@ -134,4 +135,20 @@ std::shared_ptr<arrow::Schema> make_experimental_schema(const type& t);
 /// @returns An arrow representation of `t`.
 std::shared_ptr<arrow::DataType> make_experimental_type(const type& t);
 
+/// Converts a VAST `type` to an Arrow `Field`.
+//  @param name The field name.
+/// @param t The type to convert.
+/// @returns An arrow representation of `t`.
+std::shared_ptr<arrow::Field>
+make_experimental_field(const record_type::field_view& field);
+
+/// Converts an Arrow `Schema` to a VAST `type`.
+/// @param arrow_schema The Arrow schema to convert.
+/// @returns A VAST type representation of `arrow_schema`.
+type make_vast_type(const arrow::Schema& arrow_schema);
+
+/// Converts an Arrow `DataType` to a VAST `type`
+/// @param arrow_type The arrow type to convert.
+/// @return A VAST type representation of `arrow_field`
+type make_vast_type(const std::shared_ptr<arrow::DataType>& arrow_type);
```

Comment:
Nit: Add one empty line below.

Also, does this function need to get a `const std::shared_ptr<arrow::DataType>&`, or does a `const arrow::DataType&` also suffice?

Reply:
fixed, thanks